dev/FHIR3-0-2-Geboortezorg/_LoadResources/Zwangerschapskaart-load-resources-purgecreateupdate-xml.xml

```diff
@@ -1089,7 +1089,7 @@
       <autocreate value="false"/>
       <autodelete value="false"/>
       <resource>
-         <reference value="../Zwangerschapskaart/_reference/Vrouw1-Kaart2/medmij-gbz-bc-ObstetricProcedure236992007-d735d451-2062-11ec-2089-020000000000.xml"/>
+         <reference value="../Zwangerschapskaart/_reference/Vrouw1-Kaart2/medmij-gbz-bc-ObstetricProcedure114961000146100-d735d451-2062-11ec-2089-020000000000.xml"/>
```

Comment:
Klopt het dat dit aangepast moest worden?

Reply:
Klopt, was nog een foutje in het upload script die ook gefixt moest worden